src/renderer/components/utils/SameFileImportManager.tsx

```diff
@@ -18,42 +18,58 @@ import { OpdsPublicationView } from "readium-desktop/common/views/opds";
 import { withApi } from "./api";
 
 import { ImportState } from "readium-desktop/common/redux/states/import";
+import { Download } from "readium-desktop/renderer/redux/states/download";
 
 interface Props  {
     lastImport?: ImportState;
     displayImportDialog?: any;
     search?: any;
     searchResult?: any;
     importOpdsEntry?: any;
+    downloads?: Download[];
```

Comment:
Is the type of the `importOpdsEntry` Prop:
`(data: ImportOpdsPublication) => Promise<PublicationView>`
?
If so, would it be possible to replace `any`?